contracts/interfaces/ICurve.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.10;
+pragma experimental ABIEncoderV2;
+
+interface ICurve {
+    function add_liquidity(uint256[2] memory amounts, uint256 minAmount) external payable returns (uint256);
+
+    function remove_liquidity_one_coin(
+        uint256 _token_amount,
+        int128 i,
+        uint256 _minAmount
+    ) external returns (uint256);
```

Comment:
I don't think we need those functions.

Reply:
Why using `ABIEncoderV2` in this file ?

Reply:
This contract is from the StakeDao templates. I don't think it's a good idea to have a different contract here than what's in the `stakedao-master` branch in the perp vault templates repo.

Reply:
I guess we are always only exposing functions that we use in interfaces, it is bit confusing to see all of them and it mislead that we use those too also.

Reply:
Yeah I agree. It would be nice to have a separate repo for the pricers (or have this be a monorepo) and have the pricer contracts packaged and imported for deployment to minimize redundancy.

Reply:
I'm ok keeping or removing. It def is cleaner to remove.